src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DeprecationControl.xaml

```diff
@@ -0,0 +1,83 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.DeprecationControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             mc:Ignorable="d"
+             d:DesignHeight="150" d:DesignWidth="350"
+             Background="{DynamicResource {x:Static nuget:Brushes.InfoBackgroundKey}}">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <nuget:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </UserControl.Resources>
+  <UserControl.CommandBindings>
+    <CommandBinding
+      Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}"
+      Executed="ExecuteOpenExternalLink" />
+  </UserControl.CommandBindings>
+  <Grid>
+    <Expander
+      Margin="0,12,0,16"
+      IsExpanded="True"
+      Foreground="{DynamicResource {x:Static nuget:Brushes.UIText}}">
+      <Expander.Header>
+        <TextBlock
+        FontWeight="Bold">
+          <Run
+            FontWeight="Bold"
+            Text="{x:Static nuget:Resources.Label_Deprecated}" />
+          <Run Text="{Binding Path=PackageDeprecationReasons}" />
+        </TextBlock>
+      </Expander.Header>
+      <StackPanel
+        Orientation="Vertical">
+        <TextBlock
+          Margin="0,8,0,0"
+          TextWrapping="WrapWithOverflow"
+          TextTrimming="WordEllipsis"
+          Text="{Binding Path=PackageDeprecationReasonsExplained}" />
+        <StackPanel
+          Margin="0,8,0,0"
+          Orientation="Vertical"
+          Visibility="{Binding Path=PackageDeprecationAlternatePackageLinkText, Converter={StaticResource NullToVisibilityConverter}}">
+          <TextBlock
+            Text="Alternate package"
+            FontWeight="Bold"/>
+          <StackPanel
+            Orientation="Horizontal"
+            VerticalAlignment="Center">
+            <TextBlock
+              TextWrapping="WrapWithOverflow"
+              VerticalAlignment="Center">
+              <Hyperlink
+                NavigateUri="{Binding Path=PackageDeprecationAlternatePackageLinkUrl}"
+                Style="{StaticResource HyperlinkStyle}"
+                Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}">
```

Comment:
Why are we opening the deprecated package in the browser?

Why aren't we opening it in the right pane of the PM UI. This seems like a questionable assumption that the user wants to take the next action in the browser.

Reply:
I'd actually propose to leave the hyperlink/action out of the MVP and turn this into a `TextBlock` until we actually defined an action for it (as there's none defined in the spec). @anangaur?

Reply:
Leaving it out of the MVP gives us time to consider what we want to be actionable here, and removes the whole `AlternatePackageDetailsUrl` debate for now as it wouldn't be needed at this point, if ever :)

Reply:
Yep, sounds good.

Reply:
/cc @anangaur 

Converted to TextBlock, the MVP of this feature would look like this:

![image](https://user-images.githubusercontent.com/880728/59913769-1f326380-9419-11e9-908a-04d61ccd44ff.png)